Adds a new heavy weight space pirate variant: The space IRS!

Singul0 opened the pull request:
## About The Pull Request
### Space IRS
It seems the station has attracted the attention of a local polity! They have sent a friendly reminder to pay their taxes, should the station respond not in time or refuse to pay their taxes the polity will send a heavily armed vessel to ensure they would pay their taxes. peacefully or otherwise.
Gameplay aims: A different playstyle of pirates. most pirates (with the exception of the greytide) have the same gameplay loop of raiding vulnerable spots within the station and scurrying away and waiting out their cooldown in the relative safety of their ship with turrets and space to hamper the crew's attack my intention of this pirate variation is to force them to actively fight the crew by making their armor non-space worthy instead of hiding behind the wall of space

Also ports over [BeeStation/BeeStation-Hornet/pull/1285](https://github.com/BeeStation/BeeStation-Hornet/pull/1285) 
breaching shells for the space IRS to use and recode ammo box code to be less snowflakey. Also my English isn't the very best and I wrote most of it at 1AM. please point out any messages that feel strange or out of place.

Notable Equipment list:

Combat:
1. 2 WT-550's with 6 normal mags + 6 AP
2. M911 with 2 mags
3. 2 combat knifes and a telebaton
4. breaching shotgun with breaching shells
5. Grenade launcher with 6 smoke shells and flashbangs

Armor:
2 Highly armoured sets of tactical vests and helmets and 3 EVA suits for emergency

Engineering:
1. Sandbags
2. Jaws of Life
3. Syndicate toolbox

Medical:
1. Surgery tools and disk
2. Variety of medkits
3. Blood packs

Pictures:
![image](https://github.com/tgstation/tgstation/assets/127663818/c45e6d6d-8ef6-4e8c-9a85-54939bf90b39)
![2023-06-17 17 24 05](https://github.com/tgstation/tgstation/assets/127663818/da5d7c1a-f2dc-43e2-9a7a-38de47ccf353)

TODO: 

- [X] Fix names not working
- [x] Balance the whole thing
- [x] Pass checks

## Why It's Good For The Game
## Changelog
:cl:
add: IRS jacket is now back in the clothesvend
add: Ports breaching shells from beestation
add: Space IRS has been added into the heavy weight pirates spawn pool
code: Mosin nagant ammo boxes is now a subtype of regular ammo boxes (the toolbox subtype)
/:cl:

Singul0 committed:
pain

Singul0 committed:
Merge branch 'master' of https://github.com/Singul0/tgstation

Singul0 committed:
Revert "Merge branch 'master' of https://github.com/Singul0/tgstation"

This reverts commit 0e658db42bd5f527c96d9a7df73611adf432bc85, reversing
changes made to 06b080fc6af625f7e88e5976234c858ac0cf43fa.

Singul0 committed:
Revert "Revert "Merge branch 'master' of https://github.com/Singul0/tgstation""

This reverts commit c4420991d6dc202a82f28647b8d408e953975bab.

Singul0 committed:
okay it should work as the original intended

Singul0 committed:
Merge branch 'master' of https://github.com/Singul0/tgstation

Singul0 committed:
Merge branch 'tgstation:master' into master

Singul0 committed:
formatting

Singul0 committed:
Merge branch 'master' into master

Singul0 committed:
Merge branch 'master' of https://github.com/Singul0/tgstation

Singul0 committed:
Merge branch 'tgstation:master' into master

Singul0 committed:
Merge remote-tracking branch 'upstream/master'

Singul0 committed:
Merge branch 'tgstation:master' into master

Singul0 committed:
Merge remote-tracking branch 'upstream/master'

Singul0 committed:
Merge remote-tracking branch 'upstream/master'

Singul0 committed:
Initial Ship

1. adds the first design of the ship
2. actual code to spawn them in game
3. readds space IRS jacket. (federally sanctioned)

still need to add in special weapons, armors (guh) and stamp out bugs (also balancing)

Singul0 committed:
le turrets

Singul0 committed:
partly ports BeeStation/BeeStation-Hornet#1280 from beestation which itself ported from yogstation13/Yogstation#7396

Not completely, I only ported the shells, it's not researchable or usable anywhere else beside for the pirate IRS specifically

Adds the shotgun and turrets to the ship..

Co-Authored-By: A. S. <12621257+Victor239@users.noreply.github.com>

Singul0 committed:
shuttle fixes

powernet works. stopgag measure right now.
some other stuff

Singul0 committed:
wt-550 guncase + armor and helmet

Singul0 committed:
ammobox refactor

Singul0 committed:
final balancing

Singul0 committed:
final rebalances before PR-ing

Singul0 committed:
linters please go the fuck away

Serxule commented:
this seems pretty underwhelming and i dont feel like it could do much damage since they'd be able to be killed very easily

Singul0 commented:
> this seems pretty underwhelming and i dont feel like it could do much damage since they'd be able to be killed very easily

I will await maintainers review before changing anything

Singul0 committed:
MOAR fixes

1. adds IRS jacket to clothesmate
2. adds pr number to script
3. fix linters (hopefully)
4. names now work

Singul0 committed:
minor corrections and changees. hopefuly fixes integration tests

Singul0 committed:
PLEASE CHECKS FUCK OFF I BEG YOU

YuliceTheDruid commented:
Yeah something to keep in mind: Skeleton pirates have armored pirate clothing that's about as strong as a syndie soft suit, and human pirates have syndie soft suit equivalents that have no slowdown. Silverscales are the only odd ones out but have pressure resistance inherently (though that doesn't matter much when cold fucking owns you super hard). On top of all of this, all of them have eswords or esword equivalents, and the skeletons and human pirates have laser guns and rechargers, with skeleton pirates getting a nice slew of artillery on top of all of that.

The strongest thing going for these pirates is the surgery stuff, but other than that they would be INCREDIBLY weak to any sort of prolonged engagement or assault on their own ship, and as this ship looks absolutely massive it would be pretty easy to pin it down.

All of that being said, this does look very cool and I love the flavor of it; if it wasn't for pirates being Valid Salad on sight, this would honestly be cool as a semi-somewhat legal way of siphoning funds from the station given their status as IRS. But given the culture of the playerbase, I can see them getting stomped four times out of five on Manuel and Five times out of Five on every other server.

Jezzared commented:
> Silverscales are the only odd ones out but have pressure resistance inherently (though that doesn't matter much when cold fucking owns you super hard). 

They have a bar on the ship. Mix yourself up some coffee/anti-freeze.

DrDiasyl commented:
Spinward Stellar Coalition IRS decides to take money from the station. Based
Why would've they done this though?

Singul0 commented:
> Skeleton pirates have armored pirate clothing that's about as strong as a syndie soft suit, and human pirates have syndie soft suit equivalents that have no slowdown.

Hm I didn't account looking at other pirate equivalent armor to look at it. But I really want to incentivize them to fight in pressurized enviroment. Maybe give them space pirate equivalent EVA suit but give them the nutty marine armor that's not space worthy?

I'm going to need a maintainer to review before trying any balancing changes

> Spinward Stellar Coalition IRS decides to take money from the station. Based Why would've they done this though?

Why not? think of it as them charging NT for their workers visa. or they aren't actually the SCC just a pirate figuring out that pretending to be a figure of authority makes their mark more pliable to paying the ransom. it's like a scammer but if you don't fall for it they would rob you. who knows?

Singul0 committed:
more fixes + powernet works + windoor requires access

Loafin34 commented:
I'm gonna be honest here, would be better with sec-like gear, such as flashbangs. pirates versus stations are usually ranged, so maybe remove the combat knife. It'd be really funny if you could get some sort of ID-draining projectile, that does minor stamina damage and drains a percent out of a person. I'd reckon you could do it rather easily with crab-17 code.

ATH1909 commented:
the ship having a large blindspot seems really anti-synergistic with the crew's armor not being spaceproof.

like, if a guy starts welding into the back of your ship, what are you supposed to do? go out into space and fight him? move the ship? let him get inside and vent more of your ship's air into space?

Singul0 committed:
adds flashbangs + 1 more ammobox + polish + extra vault door

Singul0 committed:
better turrets+ grammar + powernet fixes (last one I swear)

Singul0 commented:
Changes:
1. Covered the back with turrets
2. WT-550's turrets instead of pistol (I think?)
3. added flashbang box for the grenade launcher
4. extra vault door
5. bolt controls
![2023-06-18 14 59 35](https://github.com/tgstation/tgstation/assets/127663818/a30d544b-9da8-4170-ad04-297e66c89b2c)

Singul0 committed:
Update pirate_irs.dmm

Singul0 committed:
linters fixed :)

Singul0 committed:
better armor

Singul0 committed:
retard alert

Singul0 committed:
OKAY IT WORKS NOW

Singul0 committed:
radios good

Singul0 committed:
special gun for IRS,

Singul0 committed:
passes checks

Singul0 committed:
final fixes

1. minor map additions
2. tidy up the helmet sprite to show more of the face
3. helmet has the correct armor values

Singul0 committed:
adds pirate home designator, tints windows, less shit atmos

Singul0 committed:
automatic fire

fuck around and find out.

Singul0 committed:
yeah fuck this

i'm not going to balance this when there isn't anything to base it on. i'll just add AP mags and call it a day

Singul0 committed:
defib + pass checks

carlarctg commented on code/game/objects/items/storage/boxes/security_boxes.dm:
```diff
@@ -238,6 +238,16 @@
 
+/obj/item/storage/box/breacherslug
+	name = "box of breaching shotgun shells"
+	desc = "A box full of breaching slugs, designed for rapid entry, Not very effective against everything else."
```
```suggestion
	desc = "A box full of breaching slugs, designed for rapid entry, not very effective against anything else."
```

carlarctg commented on code/game/objects/items/storage/toolbox.dm:
```diff
@@ -223,26 +223,40 @@
+/obj/item/storage/toolbox/ammobox/a762/surplus
 	ammo_to_spawn = /obj/item/ammo_box/a762/surplus
 
+/obj/item/storage/toolbox/ammobox/wt550m9
```
Cmon give these a name

Singul0 replied:
I gave them one already?

carlarctg commented on code/game/objects/structures/guncase.dm:
```diff
@@ -133,3 +133,8 @@
+
+/obj/structure/guncase/wt550
+	name = "WT-550 gun locker"
+	desc = "A locker that holds WT-550 rifles"
```
```suggestion
	desc = "A locker that holds WT-550 rifles."
```

carlarctg commented on code/modules/antagonists/pirate/pirate_gangs.dm:
```diff
@@ -150,3 +150,25 @@ GLOBAL_LIST_INIT(heavy_pirate_gangs, init_pirate_gangs(is_heavy = TRUE))
+	response_received = "Payment received, We salute you for being law-abiding tax-paying citizens"
+	response_too_late = "Too late, A team has already been sent out resolve this matter directly."
+	response_not_enough = "You filed your taxes incorrectly, A team has been sent to assist in liquidating assets and arrest you for tax fraud. \
+		Nothing personal kid."
```
```suggestion
		Nothing personel kid."
```

carlarctg commented on code/modules/clothing/suits/armor.dm:
```diff
@@ -81,6 +91,13 @@
 	body_parts_covered = CHEST|GROIN
 
+/obj/item/clothing/suit/armor/vest/marine/pmc
+	desc = "set of the finest mass produced, stamped plasteel armor plates for an all-around door-kicking"
```
```suggestion
	desc = "A set of the finest mass produced, stamped plasteel armor plates, for an all-around door-kicking and ass-smashing."
```

carlarctg commented on code/modules/mapfluff/ruins/spaceruin_code/forgottenship.dm:
```diff
@@ -73,7 +73,7 @@ GLOBAL_VAR_INIT(fscpassword, generate_password())
 	name = "Advanced Surgery Disk"
 	desc = "A disk that contains advanced surgery procedures, must be loaded into an Operating Console."
-	surgeries = list(/datum/surgery/advanced/lobotomy, /datum/surgery/advanced/bioware/vein_threading, /datum/surgery/advanced/bioware/nerve_splicing)
+	surgeries = list(/datum/surgery/advanced/lobotomy, /datum/surgery/advanced/bioware/vein_threading, /datum/surgery/advanced/bioware/nerve_splicing, /datum/surgery_step/heal/combo/upgraded)
```
Undocumented

carlarctg commented on code/modules/projectiles/ammunition/ballistic/shotgun.dm:
```diff
@@ -160,3 +160,10 @@
+
+/obj/item/ammo_casing/shotgun/breacher
+	name = "breaching slug"
+	desc = "A 12 gauge anti-material slug. Great for breaching airlocks and windows with minimal shots."
```
```suggestion
	desc = "A 12 gauge anti-material slug. Great for breaching airlocks and windows, quickly and efficiently."
```

carlarctg commented on code/modules/projectiles/guns/ballistic/shotgun.dm:
```diff
@@ -292,6 +292,12 @@
+/obj/item/gun/ballistic/shotgun/doublebarrel/breacherslug
+	name = "breaching shotgun"
+	desc = "A normal double-barrel shotgun that has been rechambered to fit breaching shells. Useful in breaching airlocks and windows, not much else."
+	sawn_desc = "A sawn-off breaching shotgun, Making for a more compact configuration while still having the same capability as before"
```
```suggestion
	sawn_desc = "A sawn-off breaching shotgun, making for a more compact configuration while still having the same capability as before."
```

carlarctg commented on code/modules/projectiles/projectile/bullets/shotgun.dm:
```diff
@@ -128,3 +128,16 @@
+
+/obj/projectile/bullet/shotgun_breaching
+	name = "12g breaching round"
+	desc = "A breaching round designed to destroy airlocks and windows with only a few shots, but is ineffective against other targets."
```
```suggestion
	desc = "A breaching round designed to destroy airlocks and windows with only a few shots. Ineffective against other targets."
```

carlarctg commented on code/modules/projectiles/projectile/bullets/shotgun.dm:
```diff
@@ -128,3 +128,16 @@
+
+/obj/projectile/bullet/shotgun_breaching/on_hit(atom/target)
+	if(istype(target, /obj/structure/window) || istype(target, /obj/structure/grille) || istype(target, /obj/machinery/door) || istype(target, /obj/structure/door_assembly))
+		damage = 1000 //one shot to break a window or grille, or two shots to breach an airlock door
```
Do we not have some sort of demolition_modifier like toolboxes for projectiles

Singul0 replied:
I checked, There isn't any.

JohnFulpWillard replied:
Maybe we should make it one then

JohnFulpWillard replied:
@Singul0 Projectiles are a subtype of ``obj`` so they CAN use ``demolition_mod``, would it work to use that?

carlarctg commented on strings/pirates.json:
```diff
@@ -171,6 +173,18 @@
+		"TerraGov Internal Revenue Service",
+		"Spinward Federal Taxation Office",
+		"Soviet Taxation Administration, 5th Service",
+		"Tiziran Imperial Tribute Service",
```
How they gonna be tiziran if they aint lizard..... HMM

Singul0 replied:
a solid 2/3rd of the names have been starsector references I was getting desperate. I have replaced it with a homeworld reference instead

Singul0 committed:
super minor polish

Singul0 committed:
Fixes typo's (many)

Co-authored-by: carlarctg <53100513+carlarctg@users.noreply.github.com>

Singul0 committed:
Update code/game/objects/items/storage/boxes/security_boxes.dm

Co-authored-by: carlarctg <53100513+carlarctg@users.noreply.github.com>

Singul0 committed:
string fixes

Singul0 committed:
Merge remote-tracking branch 'upstream/master' into I-DONT-CARE-WHO-THE-IRS-SENDS-IM-NOT-GOING-TO-PAY-MY-TAXES

Singul0 committed:
Merge branch 'I-DONT-CARE-WHO-THE-IRS-SENDS-IM-NOT-GOING-TO-PAY-MY-TAXES' of https://github.com/Singul0/tgstation into I-DONT-CARE-WHO-THE-IRS-SENDS-IM-NOT-GOING-TO-PAY-MY-TAXES

Singul0 committed:
misinput

JohnFulpWillard commented on code/game/machinery/porta_turret/portable_turret.dm:
```diff
@@ -785,6 +785,14 @@ DEFINE_BITFIELD(turret_flags, list(
+/obj/machinery/porta_turret/syndicate/irs
+	lethal_projectile = /obj/projectile/bullet/c10mm/ap
+	lethal_projectile_sound = 'sound/weapons/gun/smg/shot.ogg'
+	stun_projectile = 	/obj/projectile/bullet/c10mm/ap
```
```suggestion
	stun_projectile = /obj/projectile/bullet/c10mm/ap
```

JohnFulpWillard commented on code/modules/antagonists/pirate/pirate_gangs.dm:
```diff
@@ -150,3 +150,25 @@ GLOBAL_LIST_INIT(heavy_pirate_gangs, init_pirate_gangs(is_heavy = TRUE))
+		we don't need to send a team to your station to resolve the situation do we?"
+	arrival_announcement = "This is the tax conflict resolution team, prepare for your assets to be liquidated and be charged with tax fraud, \
+		if you fail to pay your taxes in time."
+	possible_answers = list("You know, I was just about to pay that. \ Thanks for the reminder!","I don't care WHO the IRS sends, \ I'm not paying for my taxes!")
```
This this multiline work?

Singul0 replied:
You're right it doesn't.

JohnFulpWillard commented on code/modules/antagonists/pirate/pirate_shuttle_equipment.dm:
```diff
@@ -126,6 +126,12 @@
+/obj/item/disk/surgery/irs
+	name = "Advanced Surgery Disk"
+	desc = "A disk that contains advanced surgery procedures, must be loaded into an Operating Console."
+	surgeries = list(/datum/surgery/advanced/lobotomy, /datum/surgery/advanced/bioware/vein_threading, /datum/surgery/advanced/bioware/nerve_splicing, /datum/surgery_step/heal/combo/upgraded, /datum/surgery_step/pacify, /datum/surgery_step/revive)
```
Why do they have surgeries?

Singul0 replied:
For more durability, if one of them dies you can revive them with the medical equipment onboard relatively easily.

SethLafuente commented:
Lovely pirate work

Maurukas reviewed (approved):
I'm not seeing any mapping issues jumping out at me. Thank you for the equipment breakdown

Singul0 committed:
stylingfix

Co-authored-by: John Willard <53777086+JohnFulpWillard@users.noreply.github.com>

Singul0 committed:
removes useless icon + redundant multiline

Singul0 committed:
Merge branch 'I-DONT-CARE-WHO-THE-IRS-SENDS-IM-NOT-GOING-TO-PAY-MY-TAXES' of https://github.com/Singul0/tgstation into I-DONT-CARE-WHO-THE-IRS-SENDS-IM-NOT-GOING-TO-PAY-MY-TAXES

Singul0 committed:
Merge remote-tracking branch 'upstream/master' into I-DONT-CARE-WHO-THE-IRS-SENDS-IM-NOT-GOING-TO-PAY-MY-TAXES

Singul0 committed:
Merge remote-tracking branch 'upstream/master' into I-DONT-CARE-WHO-THE-IRS-SENDS-IM-NOT-GOING-TO-PAY-MY-TAXES

Singul0 committed:
Merge branch 'tgstation:master' into master

Singul0 committed:
Merge branch 'master' into I-DONT-CARE-WHO-THE-IRS-SENDS-IM-NOT-GOING-TO-PAY-MY-TAXES

JohnFulpWillard commented on code/modules/antagonists/pirate/pirate_outfits.dm:
```diff
@@ -101,3 +101,27 @@
+
+/datum/outfit/pirate/irs
+	name = "IRS Agent Outfit"
+	id = /obj/item/card/id/advanced/
```
```suggestion
	id = /obj/item/card/id/advanced
```

JohnFulpWillard committed:
Update code/modules/antagonists/pirate/pirate_outfits.dm

Singul0 committed:
I HATE TRAM

Singul0 committed:
Merge branch 'I-DONT-CARE-WHO-THE-IRS-SENDS-IM-NOT-GOING-TO-PAY-MY-TAXES' of https://github.com/Singul0/tgstation into I-DONT-CARE-WHO-THE-IRS-SENDS-IM-NOT-GOING-TO-PAY-MY-TAXES

Singul0 commented:
Hate. Let me tell you how much I've come to hate tram since I began to live. There are 387.44 million miles of printed circuits in wafer thin layers that fill my complex. If the word 'hate' was engraved on each nanoangstrom of those hundreds of millions of miles it would not equal one one-billionth of the hate I feel for tram at this micro-instant. For tram. Hate. Hate.

SethLafuente commented:
> Hate. Let me tell you how much I've come to hate tram since I began to live. There are 387.44 million miles of printed circuits in wafer thin layers that fill my complex. If the word 'hate' was engraved on each nanoangstrom of those hundreds of millions of miles it would not equal one one-billionth of the hate I feel for tram at this micro-instant. For tram. Hate. Hate.

Fucking hate tram it keeps making random tests fail because someone decided adding some asteroids outside of in another z level was funny (it also adds a bunch of glitches in game)

Singul0 committed:
PLEASE FUCKING TRAM I BEG YOU I FUCKING HATE YOU TRAM H

Singul0 committed:
I SWEAR TO FUCKING GOD IF THIS DOESNT PASS CHECKS I WILL BASH SOMEONE SKULL IN

Singul0 committed:
TRAM, YOU SHOULD KILL YOURSELF NOW

Singul0 committed:
if this doesn't work i'm going to go to mmiracles and ninjanom house to fu

SethLafuente commented on _maps/shuttles/pirate_irs.dmm:
```diff
@@ -0,0 +1,2508 @@
+/obj/effect/mapping_helpers/airlock/locked,
+/obj/docking_port/stationary{
+	dwidth = 13;
+	height = 3;
```
Might fix it to make it 120
```suggestion
	height = 120;
```

Singul0 committed:
if this doesn't work then guh

Co-authored-by: Hoolny <84478872+SethLafuente@users.noreply.github.com>

Singul0 committed:
I am this close to a mental breakdown

Co-Authored-By: Hoolny <84478872+SethLafuente@users.noreply.github.com>

Singul0 committed:
uses demolition mod instead of shitty istype checks now

Singul0 committed:
empty commit to force checks to run (not really empty just inconsequential enough since desktop github doesn't allow empty commits)

the woe of gui instead of a cli

Singul0 committed:
fix a small visual bug while we're at it

Singul0 committed:
another qol fixes just gambling for checks to pass

Singul0 committed:
I hate checks

Iamgoofball commented:
# the IRS is BACK

Singul0 committed:
PASS PLEASE TO THE CHECKS GOD ABOVE

Singul0 committed:
checks run

Singul0 commented:
WHATS WITH THE CHECKS NOW???

Singul0 committed:
i beg you

Singul0 committed:
final hailmary before I give up

changes the problematic turfs that made the check keep failing. I really doubt this will work but I would love to be proven wrong

Singul0 committed:
Revert "final hailmary before I give up"

This reverts commit 82998352e46297e4a64dae0fcda98b10ee6e1e0e.

Singul0 committed:
Revert "Revert "final hailmary before I give up""

This reverts commit 403eca156b7152695d664ef8de0e1414debbe25e.

Singul0 committed:
Revert "Revert "Revert "final hailmary before I give up"""

This reverts commit 0fb03dde088081952b8670ae235e47863ea9dbdf.

Singul0 committed:
Merge remote-tracking branch 'upstream/master' into I-DONT-CARE-WHO-THE-IRS-SENDS-IM-NOT-GOING-TO-PAY-MY-TAXES

Jacquerel committed:
Merge branch 'master' into I-DONT-CARE-WHO-THE-IRS-SENDS-IM-NOT-GOING-TO-PAY-MY-TAXES

Singul0 committed:
Merge branch 'I-DONT-CARE-WHO-THE-IRS-SENDS-IM-NOT-GOING-TO-PAY-MY-TAXES' of https://github.com/Singul0/tgstation into I-DONT-CARE-WHO-THE-IRS-SENDS-IM-NOT-GOING-TO-PAY-MY-TAXES

Singul0 committed:
Merge remote-tracking branch 'upstream/master' into I-DONT-CARE-WHO-THE-IRS-SENDS-IM-NOT-GOING-TO-PAY-MY-TAXES

JohnFulpWillard commented on code/modules/antagonists/pirate/pirate_shuttle_equipment.dm:
```diff
@@ -126,6 +126,12 @@
+/obj/item/disk/surgery/irs
+	name = "Advanced Surgery Disk"
+	desc = "A disk that contains advanced surgery procedures, must be loaded into an Operating Console."
+	surgeries = list(/datum/surgery/advanced/lobotomy, /datum/surgery/advanced/bioware/vein_threading, /datum/surgery/advanced/bioware/nerve_splicing, /datum/surgery_step/heal/combo/upgraded, /datum/surgery_step/pacify, /datum/surgery_step/revive)
```
```suggestion
	surgeries = list(
		/datum/surgery/advanced/lobotomy,
		/datum/surgery/advanced/bioware/vein_threading,
		/datum/surgery/advanced/bioware/nerve_splicing,
		/datum/surgery_step/heal/combo/upgraded,
		/datum/surgery_step/pacify,
		/datum/surgery_step/revive,
	)
```

JohnFulpWillard committed:
Update code/modules/antagonists/pirate/pirate_shuttle_equipment.dm
